src/DrupalCI/Plugin/BuildTypes/simpletest/SimpletestBuild.php

```diff
@@ -101,7 +100,7 @@ class SimpletestBuild extends Build {
    */
   public $_availableArguments = array(
     // ***** Variables Available for any build type *****
-    'DCI_UseLocalCodebase' => 'Used to define a local codebase to be cloned (instead of performing a Git checkout)',
+    'DCI_UseLocalCodebase' => 'Used to define a local codebase to be cloned (instead of performing a Git checkout)', #Covered
```

Comment:
Everything in the BuildTypes directory is only hanging around so that we can re-use these descriptions in the annotations. Once we've got the descriptions in the annotations we can get rid of these classes entirely.  Basically we're just saving them so we dont have to re write the descriptions.